Dockstore.cwl

```diff
@@ -99,7 +99,7 @@ doc: |
     - Docker (1.12.6): follow instructions to install Docker https://docs.docker.com/engine/installation
     - CWL tool
     ```
-    pip install cwltool==1.0.20170217172322
+    pip install cwltool==1.0.20180116213856
```

Comment:
I'm assuming this is the version you're testing with?

Reply:
correct